### ISSUE TITLE
self.ecx instead of self.edx

### ISSUE THREAD

**Issue by montyly** [trailofbits/binjascripts#1]
Hi,
There are two similar small typos in signed_analysis.py, at line [130](https://github.com/trailofbits/abstractanalysis/blob/master/signed_analysis.py#L130) and [143](https://github.com/trailofbits/abstractanalysis/blob/master/signed_analysis.py#L143):

```
(self.ecx).alter_current_abstract_domain(val)
```

and

```
return (self.ecx).get_current_abstract_domain()
```

(self.ecx) should probably be (self.edx).

Best,

**Comment by quend** [comment]
Good catch, thanks for letting me know.

**Comment by quend** [comment]
fixed with commit a3d553a87d0461a2664be32352e255fb57df269b
